src/lib/ActionPrototype.js

```diff
@@ -1,29 +1,57 @@
-var fetcher = require('../lib/jsonFetcher');
-var AppDispatcher = require('../lib/ep-dispatcher');
-var KeyMirror = require('keymirror');
+import fetcher from '../lib/jsonFetcher'
+import AppDispatcher  from '../lib/ep-dispatcher'
+import KeyMirror from 'keymirror'
+
+//TODOS: 
+//--turn into a class. 
+//--make it a factory function like we do with store prototype. 
 
 var ActionPrototype = {
   specialTypes: KeyMirror({
     GOT_MSG: null
   }),
+  originalActions: {},
+  stubForTests: function() {
+    this.recievedData = {}
+    let actions = Object.getOwnPropertyNames(this).filter( (fn)=> {return typeof this[fn] === 'function' } )
```

Comment:
@BJK I think we should use the spacing pattern for arrow functions that seems to be the standard most users have settled on (For example, the [Babel ES2015 overview](https://babeljs.io/docs/learn-es2015/)). Basically: no space before the argument--or the opening of the parentheses that surround the list of arguments--and the actual `=>` should have a space on each side.

Additionally, there is a space missing after the opening `{` in this call.